SCAUnitTests.cpp

```diff
@@ -0,0 +1,114 @@
+#include "pch.h"
+#include "CppUnitTest.h"
+#include "../Senior Project/SCA.h"	//FILE PATH WILL CHANGE?
```

Comment:
Yes. This path will change. I am going to merge the recent pull request by Matthew where new dir structure will be implemented. Run "Branch -> Update from main".